### PR TITLE
Adding multiple devices to boundaries exemples

### PR DESCRIPTION
**Pull request by bninassi**
adding End user devices, IoT devices and Edge devices to boundaries, according to issue#267

### CODE REVIEW

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 1]
```diff
@@ -163,6 +163,9 @@ Supporting infrastructure and systems MAY include:
 - backup
 - resources to support redundancy
 - resources to support failover
+- End user devices
```
Comment (atg-abhishek) [inline Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:166]: @bninassi I would recommend adding a few examples to each of these 3 categories to make them more useful to the reader of the SCI.

**Review by atg-abhishek — CHANGES_REQUESTED** [review]
LGTM but should add some examples to each of the 3 categories to make them more useful to the reader of the SCI

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md (hunk 1 above)
Comment (Henry-WattTime) [reply Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:166]: Intended to be inclusive, overlap is ok, no need to include examples.

**Comment by Henry-WattTime** [comment]
WG: Approved

**Pull request merged**